.gitignore

```diff
@@ -19,3 +19,11 @@
 Build/Resources/Java/selenium-server-standalone*
 /"
 Upgrading.rst
+
+# Dev Settings
+!/Configuration/Development
+/Configuration/Development/*
+!/Configuration/Development/Settings.yaml
```

Comment:
Should this default `Settings.yaml` reside in `/Configuration/Development/Docker`?

Reply:
done